src/graph/entities/edge.h

```diff
@@ -83,18 +65,6 @@ int Edge_GetRelationID
 	const Edge *edge
 );
 
-// retrieve edge source node
-Node *Edge_GetSrcNode
-(
-	Edge *e
-);
-
-// retrieve edge destination node
-Node *Edge_GetDestNode
-(
-	Edge *e
-);
```

Comment:
didn't we had situations where we needed the actual src / dest nodes from a given edge ?